server/remote_cache/byte_stream_client/byte_stream_client_test.go

```diff
@@ -43,30 +50,38 @@ func TestManifest_SomeFilesZip(t *testing.T) {
 			},
 		},
 	}
-	path, _ := bazel.Runfile("some_files.zip")
-	bytes, _ := os.ReadFile(path)
-	manifest, _ := ParseZipManifestFooter(bytes, 0, int64(len(bytes)))
+	path, err := runfiles.Rlocation(someFilesRunfilePath)
+	assert.NoError(t, err)
```

Comment:
nit: use `require.NoError` for these, here and below, since these are fatal errors